fedimint-core/src/config.rs

```diff
@@ -370,6 +370,12 @@ impl FromStr for FederationIdPrefix {
     }
 }
 
+impl FederationIdPrefix {
+    pub fn as_bytes(&self) -> Vec<u8> {
```

Comment:
Idiomatically, if it allocates, it should be `to_`. `as_` is only for near-zero-cost conversions.

Reply:
Done